PxCs/Extensions/SoundExtension.cs

```diff
@@ -0,0 +1,302 @@
+using System;
+using System.Text;
+using System.IO;
+using System.Collections.Generic;
+
+namespace PxCs.Extensions
+{
+    internal enum WAVE_FORMAT
+    {
+        UNKNOWN,
+        PCM,
+        ADPCM,
+        IMA_ADPCM = 0x11,
+    }
+
+    internal class Ima
```

Comment:
Internal class. Never used it before in PxCs. But I think it's fine here, as we won't expose these classes.